src/frontend/Grammar.mly

```diff
@@ -275,6 +275,11 @@ plain_term_except_cof_case:
   | COM; fam = atomic_term; src = atomic_term; trg = atomic_term; phi = atomic_term; body = atomic_term
     { Com (fam, src, trg, phi, body) }
 
+  | left = atomic_term; DMEET; right = atomic_term   /* left and right should maybe not be atomic */
+    { DMeet (left, right) }
+  | left = atomic_term; DJOIN; right = atomic_term
+    { DJoin (left, right) }
```

Comment:
Interestingly, I get a compilation error if I switch from atomic_term to term...

Reply:
Also, I can't get nested connections to parse, i.e. `(d1 ⊓ d0) ⊔ d1`.

Reply:
> Also, I can't get nested connections to parse, i.e. `(d1 ⊓ d0) ⊔ d1`.

#69

Reply:
Ahhh...curly brackets. Why the unorthodox choice?